apps/dashboard/src/@/components/DashboardTypeCookieSetter.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
+
+import { useEffect } from "react";
+import { setCookie } from "../../stores/SyncStoreToCookies";
+
+export function DashboardTypeCookieSetter(props: {
+  type: "team" | "old";
+}) {
+  // eslint-disable-next-line no-restricted-syntax
+  useEffect(() => {
+    setCookie("x-dashboard-type", props.type);
+  }, [props.type]);
+
+  return null;
+}
```

Comment:
can we keep this server-side somehow?

Reply:
the login page does the redirect to /dashboard or /team on client side once the login is done, on the landing page - we change the link to get started based on cookie. so doing this client side is much easier.